package/MDAnalysis/analysis/rms.py

```diff
@@ -380,7 +380,10 @@ def __init__(self, atomgroup, reference=None, select='all',
              corresponding atom in `select`, and assumes ``None`` for `groupselections`.
 
         weights_groupselections : False or list of {"mass", ``None`` or array_like} (optional)
-             1. ``False`` will apply imposed weights to `groupselections` from ``weights`` option.
+             1. ``False`` will apply imposed weights to `groupselections` from
+             ``weights``option if `weights` is iterable. Otherwise will assume
```

Comment:
I think you might have this the wrong way around!

https://github.com/MDAnalysis/mdanalysis/blob/ff7ffa10901e2df2be12c3d3dd78e4e0a262e90e/package/MDAnalysis/analysis/rms.py#L573-L577